pom.xml

```diff
@@ -44,31 +44,17 @@
                 <directory>src/main/resources</directory>
                 <filtering>true</filtering>
                 <includes>
-                    <include>paper-plugin.yml</include>
+                    <include>plugin.yml</include>
                 </includes>
             </resource>
         </resources>
         <plugins>
-            <plugin>
```

Comment:
This plugin is used to fetch commit data to append it to the plugin version. Why do you remove this?